src/components/Books.jsx

```diff
@@ -1,25 +1,14 @@
 import React from 'react';
-import { v4 as uuidv4 } from 'uuid';
-import BookItems from './BookItems';
+// import { v4 as uuidv4 } from 'uuid';
+// import BookItems from './BookItems';
```

Comment:
- [ ] Just as I pointed out above about Following the best practices, you are not meant to leave unused code as a comment on your code base. I'll suggest you remove this as it makes your codebase look a bit a bit unprofessional. :+1: :smile: